components/user/UserCard.js

```diff
@@ -13,21 +14,36 @@ export default function UserCard({ profile }) {
         <FallbackImage
           src={`https://github.com/${profile.username}.png`}
           alt={`Profile picture of ${profile.name}`}
-          width={80}
-          height={80}
+          width={75}
+          height={75}
           className="rounded-full"
           fallback={profile.name}
         />
       </div>
       <div>
-        <h3 className="text-xl justify-center text-center mb-2 text-tertiary-medium font-bold">
+        <h3 className="text-lg justify-center text-center mb-2 text-tertiary-medium font-bold">
```

Comment:
Because of the decrease in font size, the color need to be changed to increase contrast on a light background.

Reply:
This is a very good point.  WDYT about the overall concept so far though?  Trying to decide if I should stick with this direction or perhaps a different iteration/style of card

Reply:
I'm not sure if there is a great way to show all tags and the cards look balanced.
I dealt with this problem on my portfolio site & never found a good solution.

If we are only showing tags that match a search term, then I think this will look OK, maybe with a accent color for the border.

Reply:
Yeah that's where my mind is going too.  Show tags based on the search terms (up to X amount so they fit) since this issue is about making the search make sense from a user perspective.  Thanks!